src/Networking/MQTT/MqttClient.h

```diff
@@ -55,32 +57,28 @@ class MqttClient : public NetworkClient
 	void ConnectionLost() noexcept override;
 	static void PublishCallback(void** state, struct mqtt_response_publish *published);
 
-	mqtt_client client;
 	uint8_t sendBuf[SendBufferSize];
 	uint8_t recvBuf[ReceiveBufferSize];
+	mqtt_client client;
 
 	Subscription *prevSub, *currSub; // Used for subscribing to topics
-	OutputBuffer *currBuf; // Current message being published
 
 	uint32_t messageTimer;	// General purpose variable for keeping track of queued messages timeout
-	Mutex publishMutex;
 
 	MqttClient *next;
```

Comment:
Under what conditions can we have more than one MqttClient?

Reply:
The case where more than one MqttClient instance is needed is for when there are multiple brokers - either on the same interface or on different interfaces. 

Per our email conversation, only one broker on one interface needs to be supported for now. So I changed MqttClient to be a singleton. This is remnant of the old code I forgot to delete.